src/screens/Technology/components/Details/index.tsx

```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
         presentationStyle="overFullScreen"
       />
 
-      {false && <S.CostSection>Vídeos</S.CostSection>}
+      {technology.attachments.documents.length ? (
+        <>
+          <S.CostSection style={{ marginTop: 30 }}>Vídeos</S.CostSection>
+
+          {videos.map((video, idx) => (
+            <View key={idx} style={{ marginBottom: 0, backgroundColor: '#ddd' }}>
+              <YoutubePlayer
+                height={240}
+                videoId={video.videoId}
+              />
+            </View>
+          ))}
+        </>
+      ) : null}
+
+      {technology.attachments.documents.length ? (
+        <>
+          <S.CostSection style={{ marginTop: 30 }}>Documentos</S.CostSection>
+          {technology.attachments.documents.map((document, idx) => (
+            <TouchableOpacity
+              key={idx}
```

Comment:
Check if `id` exists in `document` before changes, please
```suggestion
          {technology.attachments.documents.map((document, idx) => (
            <TouchableOpacity
              key={document.id}
```